web-console/knikubevirt/Create-vm/step-2-networking/create-vm-networking.md

```diff
@@ -3,17 +3,21 @@
 ## Step 2: Networking
 
 ![Default](img/3-00.png)
-The user enters the Networking step with an existing NIC pre-defined, set to Type (local). In this step, the 'Review and Create' button is still available to skip the rest of the steps. At this point, the user can select to customize this NIC, add new ones, or remove any they wish.
+The user enters the Networking step with an existing  Network interface pre-defined, set to Type (local). In this step, the 'Review and Create' button is still available to skip the rest of the steps. At this point, the user can select to customize this Network interface, add new ones, or remove any they wish.
 
 ![Adding NIC](img/3-01.png)
-Clicking on 'Add NIC' will pop up a "Create NIC" modal. The NIC name is prepopulated with a numeric count form of existing NICs on this flow. The user may change the NIC name. All the fields are preselected except Type. In order for the user to apply and add this NIC, the user must select one of the type options.
+Clicking on 'Add  Network interface' will pop up a "Create  Network interface" modal. The Network interface name is prepopulated with a numeric count form of existing  Network interfaces on this flow. The user may change the Network interface name. All the fields are preselected except Type. In order for the user to apply and add this Network interface,they must select one of the type options.
```

Comment:
we can pre-select the name, but I do not think it is a good idea for other fields

Reply:
OK, I removed the pre-selection from 'model and 'network'. 
Thanks, @suomiy!